fmriprep/workflows/bold/base.py

```diff
@@ -573,12 +573,12 @@ def init_func_preproc_wf(bold_file):
         # And ensure echo is dropped from report
         for node in fmap_unwarp_report_wf.list_node_names():
             if node.split('.')[-1].startswith('ds_'):
-                fmap_unwarp_report_wf.get_node(node).interface.out_path_base = 'fmriprep'
+                fmap_unwarp_report_wf.get_node(node).interface.out_path_base = ""
                 fmap_unwarp_report_wf.get_node(node).inputs.dismiss_entities = ("echo",)
```

Comment:
I believe these loops can go away (setting `out_path_base` bc of the class modification above, and the `dismiss_entities` because of your PR to drop repeated entities).

Reply:
These are coming from `sdcflows`, not fMRIPrep.